volumes/sql/traffic_new/func_upd_arc_link.sql

```diff
@@ -0,0 +1,38 @@
+-- FUNCTION: traffic.update_arc_link()
+
+-- DROP FUNCTION IF EXISTS traffic.update_arc_link();
+
+CREATE OR REPLACE FUNCTION traffic.update_arc_link(
+	)
+    RETURNS void
+    LANGUAGE 'sql'
+    COST 100
+    VOLATILE SECURITY DEFINER PARALLEL UNSAFE
+AS $BODY$
+
+    insert into traffic.arc_link (
+	select * from "TRAFFIC_NEW"."ARC_LINK"
+	except
+	select * from traffic.arc_link)
+	on conflict (link_id) 
+	DO UPDATE 
+		SET arc_id = EXCLUDED.arc_id,
+			frequency = EXCLUDED.frequency;
+
+with delrec as (
+	select * from traffic.arc_link 
+	except 
+	select * from "TRAFFIC_NEW"."ARC_LINK")
+
+Delete from traffic.arc_link where link_id in (SELECT link_id from delrec);
```

Comment:
I'm wondering if we're not keeping any of the old data (e.g. `traffic.arc_link`), wouldn't `TRUNCATE` and `INSERT` do the same thing faster than `UPSERT`?

Reply:
depends on whether we're auditing that table.

Reply:
I got it. So, we're auditing all these tables

Reply:
Not the really big ones, which does point to me that there should be a smol README edit to highlight which tables are being audited.